eo-maven-plugin/src/main/java/org/eolang/maven/hash/CommitHashesText.java

```diff
@@ -49,15 +47,13 @@ final class CommitHashesText extends TextEnvelope {
     /**
      * Cache.
      */
-    private static final Text CACHE = new Sticky(
-        CommitHashesText.asText(CommitHashesText.HOME)
-    );
+    private static final String CACHE = CommitHashesText.asText(CommitHashesText.HOME);
```

Comment:
@volodya-lombrozo please take a look. I think we do not need here `Sticky` at all. Note, that 
`CommitHashesText.asText(CommitHashesText.HOME)` is calling here anyway, why do we need to wrap it to `Sticky`. Also, in `CommitHashesText.asText(CommitHashesText.HOME)` `String` is already calulated but is wrapping to `Text`.